benchmarks/test_benchmark_coo.py

```diff
@@ -36,3 +52,91 @@ def test_elemwise(benchmark, f, elemwise_args):
     @benchmark
     def bench():
         f(x, y)
+
+
+@pytest.fixture(scope="module", params=[100, 500, 1000], ids=side_ids)
+def elemwise_broadcast_args(request):
+    side = request.param
+    rng = np.random.default_rng(seed=SEED)
+    if side**side >= 2**26:
+        pytest.skip()
+    x = sparse.random((side, 1, side), density=DENSITY, random_state=rng)
+    y = sparse.random((side, side), density=DENSITY, random_state=rng)
+    return x, y
+
+
+@pytest.mark.parametrize("f", [operator.add, operator.mul])
+def test_elemwise_broadcast(benchmark, f, elemwise_broadcast_args):
+    x, y = elemwise_broadcast_args
+    f(x, y)
+
+    @benchmark
+    def bench():
+        f(x, y)
+
+
+@pytest.fixture(scope="module", params=[100, 500, 1000], ids=side_ids)
+def indexing_args(request):
+    side = request.param
+    rng = np.random.default_rng(seed=SEED)
+    if side**side >= 2**26:
+        pytest.skip()
+
+    return sparse.random((side, side, side), density=DENSITY, random_state=rng)
+
+
+def test_index_scalar(benchmark, indexing_args):
+    x = indexing_args
+    side = x.shape([0])
+
+    x[5]  # Numba compilation
```

Comment:
I'd do exactly the same thing inside and outside the benchmark -- to make sure the same function is compiled.

Reply:
Could you clarify please?
I though you had said to remove this from the benchmark. 
BTW there are other issues. I'm working on that.

Reply:
Below, inside the benchmark, you are doing 
```suggestion
    x[side // 2, side // 2, side // 2]  # Numba compilation
```

So you should do the same here.